fudy/NoteScreen/Component/FramelessWindow.qml

```diff
@@ -111,15 +114,50 @@ ApplicationWindow {
 		onDoubleClicked: function() {
 			root.doubleClicked();
 		}
+
+		onPositionChanged: function(mouse) {
+			if (mouse.x >= parent.width - internal.margin && mouse.y >= parent.height - internal.margin) {
+				mouseArea.cursorShape = Qt.SizeFDiagCursor;
+			} else {
+				mouseArea.cursorShape = Qt.ArrowCursor;
+			}
+		}
 	}
 
-	ColumnLayout {
-		anchors.fill: parent
-		spacing: 0
+	Flickable {
+		id: scrollView
+		width: parent.width
+		height: parent.height - 30
+		contentWidth: parent.width
+		contentHeight: loaderLayout.implicitHeight
+		ScrollBar.vertical: ScrollBar {
+			id: scrollBar
+			hoverEnabled: true
+			active: hovered || pressed
+			orientation: Qt.Vertical
+			anchors {
+				top: parent.top
+				bottom: parent.bottom
+				left: parent.right
+				leftMargin: -20
+			}
+		}
 
-		Loader {
-			id: contentLoader
-			Layout.fillWidth: true
+		ColumnLayout {
+			id: loaderLayout
+			anchors.fill: parent
+			anchors.margins: 10
+			spacing: 0
+
+			Loader {
+				id: contentLoader
```

Comment:
Can be removed

Reply:
Please clarify what you prefer to do. Thank you.